lib/constants/custom_color.dart

```diff
@@ -11,14 +11,14 @@ const onboardHeadingColor = Color(0xFF032E9A);
 const splashHeadingColor = Color(0xFF213656);
 const onboardTextColor = Color(0xFF213656);
 
-Color lightgreenshede = Color(0xFFF0FAF6);
-Color lightgreenshede1 = Color(0xFFB2D9CC);
-Color greenshede0 = Color(0xFF66A690);
-Color greenshede1 = Color(0xFF93C9B5);
-Color primarygreen = Color(0xFF1E3A34);
-Color grayshade = Color(0xFF93B3AA);
-Color colorAcent = Color(0xFF78C2A7);
-Color cyanColor = Color(0xFF6D7E6E);
+Color lightgreenshede = const Color(0xFFF0FAF6);
```

Comment:
Shouldn't these be `const Color lightgreenshede` instead?

Reply:
Hmm, you're right. I used the auto-fix linter on vs code.
I'll fix them myself.